wheelhouse.txt

```diff
@@ -7,15 +7,16 @@ pip<22.1;python_version >= '3.8'
 # pin Jinja2, PyYAML and MarkupSafe to the last versions supporting python 3.5
 # for trusty
 Jinja2==2.10;python_version >= '3.0' and python_version <= '3.4' # py3 trusty
-Jinja2==2.11;python_version == '2.7' or python_version == '3.5'  # py27, py35
-Jinja2;python_version >= '3.6' # py36 and on
+Jinja2==2.11;python_version == '2.7' or  python_version == '3.5' # py27, py35 xenial
+Jinja2==2.11;python_version >= '3.6' and python_version <= '3.8' # py36 bionic, py38 focal
```

Comment:
Perhaps use `<` pins here (and on line 9)?

Reply:
Okay, done!